transport-classes-io_uring/src/main/java/io/netty/channel/uring/Native.java

```diff
@@ -340,10 +340,25 @@ static boolean isIOUringSupportSplice(int ringFd) {
         return ioUringProbe(ringFd, new int[] { Native.IORING_OP_SPLICE });
     }
 
+    /**
+     * check current kernel version whether support io_uring_register_io_wq_worker
+     * Available since 5.15.
+     * @return true if support io_uring_register_io_wq_worker
+     */
+    static boolean isRegisterIOWQWorkerSupported(int ringFd) {
+        // See https://github.com/torvalds/linux/blob/v5.5/fs/io_uring.c#L5488C10-L5488C16
```

Comment:
This doesn't look right...

Reply:
why ?

Reply:
I mean the link provided, not what it does from a functional pov

Reply:
hmmm I think the link is correct as it shows it will return `-EINVAL`

Reply:
I probably misread what was the point of it, sorry :)